.github/workflows/publish-nodes.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish Nodes
+on:
+  push:
+    branches:
+    - main
+  
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: use node 18
+      uses: actions/setup-node@v2
+      with:
+        node-version: '18'
+    - run: npm ci
+    - run: npm run lint
+    - run: npm run compile
+    - name: Publish to production
+      run: npm run publish:nodes
+      env:
+        NODESCRIPT_API_URL: https://api.nodescript.dev
+        NODESCRIPT_API_TOKEN: ${{ secrets.NODESCRIPT_API_TOKEN_PRODUCTION }}
```

Comment:
This variable exists in org. I think in our current plan org-level variables are permitted with public repos, so I reckon we're ok.

<img width="408" alt="image" src="https://github.com/user-attachments/assets/8825f84a-af55-4c6d-9280-09bd7ba03160">